Core/MIPS/ARM64/Arm64IRCompFPU.cpp

```diff
@@ -98,7 +98,21 @@ void Arm64JitBackend::CompIR_FAssign(IRInst inst) {
 		break;
 
 	case IROp::FSign:
-		CompIR_Generic(inst);
+		regs_.Map(inst);
+		// We'll need this flag later.  Vector could use a temp and FCMEQ.
+		fp_.FCMP(regs_.F(inst.src1));
+
+		fp_.MOVI2FDUP(EncodeRegToDouble(SCRATCHF1), 1.0f);
```

Comment:
Hm, why EncodeRegToDouble?

Reply:
MOVI is a vector instruction, which can only operate on doubles and quads.  Doesn't make sense to DUP to a single.

As to why I'm using vector, all the other ops that were convenient to use for this operation were vector, so I'm assuming there are benefits to keeping everything vector and not swapping to scalar.

-[Unknown]

Reply:
Yeah sorry I keep associating these with actual double floats, rather than half a SIMD register... Not yet enough in the mindset of ARM64 NEON :/